js/components/portal-dashboard/feedback/activity-level-feedback-student-rows.tsx

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import { Map } from "immutable";
+import { getFormattedStudentName } from "../../../util/student-utils";
+import AwaitingFeedbackActivityBadgeIcon from "../../../../img/svg-icons/awaiting-feedback-activity-badge-icon.svg";
+import GivenFeedbackActivityBadgeIcon from "../../../../img/svg-icons/given-feedback-activity-badge-icon.svg";
+import UpdateFeedbackQuestionBadgeIcon from "../../../../img/svg-icons/update-feedback-question-badge-icon.svg";
+
+import css from "../../../../css/portal-dashboard/feedback/feedback-rows.less";
+
+interface IProps {
+  activityId: string | null;
+  activityIndex: number;
+  feedbacks: Map<any, any>;
+  feedbacksNeedingReview: Map<any, any>;
+  isAnonymous: boolean;
+  updateActivityFeedback: (activityId: string, activityIndex: number, platformStudentId: string, feedback: any) => void;
+}
+
+export const ActivityLevelFeedbackStudentRows: React.FC<IProps> = (props) => {
+  const { activityId, activityIndex, feedbacks, isAnonymous, updateActivityFeedback} = props;
+
+  const handleFeedbackChange = (studentId: string) => (event: React.FormEvent<HTMLTextAreaElement>) => {
+    if (activityId && studentId != null) {
```

Comment:
studentId is passed in as a string so this test can simply be:
`if (activityId) {`